worldbank/src/Components/Networking.js

```diff
@@ -38,21 +38,33 @@ class Networking {
 
   postSearch = async (country, indicator, start) => {
     let response = await fetch(`${url}/search`, {
-      method: "POST",
+      method: "GET",
       credentials: "include",
       headers: {
         "Content-Type": "application/json",
       },
       body: JSON.stringify({
-        country: country,
-        indicator: indicator,
-        start: start,
+        CountryName: country,
```

Comment:
in the backend i have called these country, indicator and year. Could you make them match please?